tests/python/gaia-ui-tests/gaiatest/tests/functional/gallery/manifest.ini

```diff
@@ -46,9 +46,8 @@ skip-if = device == "desktop" && os == "linux" && debug
 
 [test_gallery_switch_to_camera.py]
 # Bug 1111637 - Create a new test case for the switching to camera from gallery and back
-external = true
 camera = true
-sdcard = true
```

Comment:
The answer is, the camera does start up, but with a message of some sorts that you need a storage card or something.